kubernetes_asyncio/client/rest.py

```diff
@@ -117,7 +117,16 @@ async def request(self, method, url, query_params=None, headers=None,
 
         post_params = post_params or {}
         headers = headers or {}
-        timeout = _request_timeout or 5 * 60
+        timeout = aiohttp.ClientTimeout()
+        if _request_timeout:
+            if isinstance(_request_timeout, (int, float)):
```

Comment:
Could you add a branch for a case where _request_timeout is an instance of aiohttp.ClientTimeout, like in this code? 

https://github.com/PrefectHQ/prefect/blob/55380bceb768277c9db96c8544322e41110ef9e7/src/integrations/prefect-kubernetes/prefect_kubernetes/worker.py#L869

Reply:
Ok, Added the branch to handle the _request_timeout if its ClientTimeout object.